src/App/App.tsx

```diff
@@ -35,6 +36,12 @@ export default function App() {
 
   const { isLoading, error, movies } = useLoadMovies();
 
+  const cardContextValue = {
```

Comment:
It's better to leverage useMemo and useCallback here to avoid context changes.

Reply:
Added useMemo and useCallback hooks.